core/lib/zksync_core/src/api_server/web3/state.rs

```diff
@@ -541,12 +541,9 @@ impl<E> RpcState<E> {
     }
 }
 
-/// Contains mapping from index to `Filter` with optional location.
-#[derive(Default, Debug)]
-pub(crate) struct Filters {
-    state: HashMap<U256, InstalledFilter>,
-    max_cap: usize,
-}
+/// Contains mapping from index to `Filter`x with optional location.
```

Comment:
Nit: You've added an `x` after `Filter`, probably as a typo. Also, "with optional location" part is probably outdated; at least, I don't understand what it is referring to.